enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -23,7 +23,25 @@ Features planned:
 
 # Getting started
 
-## Start a local build of Metabase from master
+## Start Metabase
+
+You have a couple of options to start Metabase:
```

Comment:
I think we need to add a sentence about why their existing MB instance won't work.
Maybe something like this

```suggestion
Currently, the SDK only works with specific versions of Metabase.
You have the following options:
```